### ISSUE TITLE
 atmospheric, hydrographic, physiographic features 

### ISSUE THREAD

**Issue by alanruttenberg** [CommonCoreOntology/CommonCoreOntologies#250]
Should these be parts of Portion of Atmosphere, Portion of Cryosphere, and Portion of Lithosphere, respectively

And should the portions be part of a "natural satellite" or "astronomical body" as discussed in https://github.com/CommonCoreOntology/CommonCoreOntologies/issues/249

**Comment by alanruttenberg** [comment]
Also, Atmospheric Feature doesn't satisfy the definition of Geographic Feature given the usual sense or the word "topographic" used in that definition. It can be pushed up to Environmental Feature.

**Comment by cameronmore** [comment]
It seems like the rationale for keeping them separate is that 'Portion of Lithosphere' can be a fiat bounding box, while Physiographic Feature picks out a definite, not fiat, feature of the environment, like a hill, valley, etc. So, in practice I can tag geographic data with the features and arbitrary polygons with the portions.

**Comment by alanruttenberg** [comment]
Seems like a bad choice to have those *have to be* fiat. I think they should be moved up to material entity, the proposed subclassing done with the understanding that some may be fiat and some object. Reinforces my own practice that making distinctions between the subclasses of material entity is counterproductive.

If I have a bottle of air, the boundaries of the air are not fiat and this would have to be called an atmospheric feature rather than portion of atmosphere, which just seems wrong.

**Comment by cameronmore** [comment]
Revisiting this topic, what is the full recommendation? To move the Portion terms out of 'fiat object part' and directly under 'material entity', and then move the three feature classes under them?

**Comment by alanruttenberg** [comment]
1. Yes move them to material entity. Definitions will need to be adjusted. Another example of the problem: Portion of cryosphere: A fiat object part of the frozen part of a natural satellite's hydosphere
**The** frozen part of the hydrosphere isn't an object - it is a disconnected set, so part of an object aggregate if anything.

2. Personally I think the feature terms should be dropped and any subs be put under the portion terms. They are problematic in that "feature" things would in general be defined in terms of a feature role. Insofar as the distinction between between sharp and fuzzy boundaries go, a valley isn't definite - having a polygon mark it involves having to decide at what elevation the cutoff is.

**Comment by APCox** [comment]
I agree that the "Portion of _____" terms should be moved directly under 'material entity' instead of 'fiat object part'.

I agree that the current definition of 'Geographic Feature':
`An Environmental Feature that is a natural (i.e. not human made) topographical feature having a (relatively) stable location in some Geospatial Region which can be designated by location-specific data.`
needs improvement and is not well-suited to asserting 'Atmospheric Feature' as a subtype.

As with 'Portion of Geosphere' (see issue #249), 'Geographic Feature' is already too limiting of a term because it is Earth-specific. Hence, according to the current definitions, there are no atmospheric, hydrographic, or physiographic features on other astronomical bodies.

The relationship between the "Portion of _____ __" and "__ _____ Feature" terms is at best confusing. I believe Cameron's first answer above is in line with the thinking in having both sets of terms. I propose revising the definitions of 'Atmospheric Feature', 'Hydrographic Feature', and 'Physiographic Feature' to make them sufficiently distinct from their "Portion of _____" counterparts. I will note that there are domain ontologies built entirely off of these 3 terms.

**Comment by dlutz2** [comment]
Agree with @alanruttenberg that the Environmental Feature terms could be deprecated and deserves a re-think.
-  Geographers/cartographers have a large collection of types, structures, point/line/area features which may/may not fit well under the current partitioning of 'Geographic Feature'
- Incorporating non-Earth features probably too much since there are more _spheres to include (Methane seas, ammonia volcanoes, and permanent/fixed cyclones rare on Earth)
- 'Anthropogenic Feature' is effectively "Material Artifacts that rarely move" which is rather broad. 
- There is a whole deep domain here. How much of that belongs in a mid-tier and how much deserves its own domain ontology?
